languages/wikipron/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flake8==3.7.9
```

Comment:
This is fine. Maybe we want to remove the versioning from requests and requests-html under the assumption they're probably stable but let's get this in and tinker later.

Dumb question also---why do we need flake8 here?

Reply:
I suppose we don't need flake8 here, but I've added it because I run it prior to committing.

Reply:
Yeah, I'd cut it. Ready to merge otherwise?

Reply:
I believe so. I'll clean up `requirements.txt` in a future PR.